xbmc/input/WindowTranslator.cpp

```diff
@@ -19,7 +19,10 @@
  */
 
 #include "WindowTranslator.h"
+#include "Application.h"
 #include "guilib/WindowIDs.h"
+#include "ServiceBroker.h"
```

Comment:
Can you put this after the folder includes?

Reply:
it should be in alphabetic order, my fault, because p comes before s 😄